app/partials/coinify/summary.pug

```diff
@@ -1,105 +1,80 @@
+  .flex-column.mbl(ng-show="error || $parent.error")
+    p.mvm.state-danger-text.type-md(translate="UNABLE_TO_PROCESS")
+    span.mvm(translate="CANNOT_COMPLETE_TRADE_ERROR" translate-values="{error: $parent.error.error_description}")
+  .flex-column.mbl(ng-hide="error || status.creatingSellTrade")
+    span.type-h5.em-500.flex-start.pts(
+      ng-hide="isBankTransfer() && summaryForm.fiat.$invalid"
+      translate="CONFIRM_YOUR_ORDER")
+    span.type-h5.em-500.flex-start.pts.state-danger-text(
+      ng-show="isBankTransfer() && summaryForm.fiat.$error.max"
+      translate="ERROR_DAILY_LIMIT")
+    span.type-h5.em-500.flex-start.pts.state-danger-text(
+      ng-show="isBankTransfer() && summaryForm.fiat.$error.min && !summaryForm.fiat.$error.max"
+      translate="ERROR_MIN_LIMIT")
+    p.mvm
+      span(translate="ORDER_CONFIRMATION_COPY" ng-if="summaryForm.fiat.$valid" ng-hide="isBankTransfer()")
+      span(translate="ORDER_CONFIRMATION_BANK_COPY" ng-if="summaryForm.fiat.$valid" ng-hide="!isBankTransfer()")
+      span(translate="ERROR_DAILY_LIMIT_COPY" ng-show="isBankTransfer() && summaryForm.fiat.$error.max" translate-values="{code: tempTrade.fiatCurrency, max: limits[vm.medium].max[tempTrade.fiatCurrency], medium: vm.medium}")
+      span(translate="ERROR_MIN_LIMIT_COPY" ng-show="isBankTransfer() && summaryForm.fiat.$error.min && !summaryForm.fiat.$error.max" translate-values="{code: tempTrade.fiatCurrency, min: limits[vm.medium].min[tempTrade.fiatCurrency], medium: vm.medium}")
+    .flex-center.flex-between.flex-row-reverse
+      .mam
+        span.type-sm.blue.underline.pointer(ng-click="state.editAmount = true" ng-hide="state.editAmount" translate="EDIT_ORDER")
+        span.type-sm.blue.underline.pointer(ng-click="state.editAmount = false" ng-show="state.editAmount" translate="CANCEL")
+      .flex-center(ng-show="(summaryForm.fiat.$error.max || summaryForm.fiat.$error.min) && !isBankTransfer()")
+        i.ti-alert.state-danger-text.mrs.type-h4
+        span.state-danger-text.mrs(ng-show="summaryForm.fiat.$error.max" translate="MAX_LIMIT_EXCEEDED" translate-values="{code: tempTrade.fiatCurrency, max: limits[vm.medium].max[tempTrade.fiatCurrency], medium: vm.medium}")
+        span.state-danger-text.mrs(ng-show="summaryForm.fiat.$error.min && !summaryForm.fiat.$error.max" translate="BELOW_MIN_LIMIT" translate-values="{code: tempTrade.fiatCurrency, min: limits[vm.medium].min[tempTrade.fiatCurrency], medium: vm.medium}")
+    .border.pam(ng-class="{'bg-grey': state.editAmount}")
+      .flex-row.flex-between.pam.border-bottom-light
+        span(ng-hide="isBankTransfer()" translate="BTC:")
+        span(ng-show="isBankTransfer()" translate="BTC_APPROX_RATE")
+        span.success(ng-hide="isBankTransfer()" uib-tooltip="{{'RECEIVE_BITCOIN_TO' | translate:{label: label} }}") {{ format(fromSatoshi(trade.BTCAmount || 0, bitcoin), bitcoin, true) }}
```

Comment:
I would suggest creating a filter, so you can do e.g. `trade.BTCAount | myMagicFormat:...:...`. It's easier to write test cases for a filter than for view code.